From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Tue, 17 May 2022 18:42:37 +0800
Subject: gdk/: Don't distribute the built sources

The Visual Studio projects have support for generating these items, so
we don't really need to distribute them with the sources.

Origin: upstream, 3.24.35, commit:abbf7a3497ec94a97f47b487dd37f4b9ffc36740
---
 gdk/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gdk/Makefile.am b/gdk/Makefile.am
index 860973b..632c687 100644
--- a/gdk/Makefile.am
+++ b/gdk/Makefile.am
@@ -372,7 +372,6 @@ lib_LTLIBRARIES = libgdk-3.la
 
 MAINTAINERCLEANFILES = $(gdk_built_sources) stamp-gdkenumtypes.h
 EXTRA_DIST += \
-	$(gdk_built_sources)	\
 	fallback-c89.c
 
 install-exec-hook:
